### PR TITLE
style(plugins-wrk): lint unused-vars

### PR DESCRIPTION
**Pull request by jesusprubio**
#### Please confirm that your PR fulfills these requirements
- [x] Multiple commits are squashed into one commit.
- [x] The commit message follows [Conventional Commits](https://github.com/IBM/kui/blob/master/CONTRIBUTING.md#conventional-commits), which allows us to autogenerate release notes; e.g. `fix(plugins/plugin-k8s): fixed annoying bugs`
- [x] All npm dependencies are pinned.

### CODE REVIEW

File: plugins/plugin-wrk/src/lib/chart.ts [hunk 1]
```diff
@@ -68,7 +65,7 @@ export const init = (graphics, options: IOptions = new DefaultOptions()) => {
   /** create a ChartJS chart configuration object */
   const makeChartConfig = () => {
     // pick up the theme choices
-    const { fontFamily } = theme(ctx)
```
Comment (myan9) [inline plugins/plugin-wrk/src/lib/chart.ts]: This doesn't seem right.

**Comment by jesusprubio** [comment]
We should until we can drop these files and enable de rule in the last one.

File: plugins/plugin-wrk/src/lib/chart.ts [hunk 2]
```diff
@@ -229,51 +210,5 @@ export const init = (graphics, options: IOptions = new DefaultOptions()) => {
...
-
-        // if (N > 0) {
-        chart.update()
-        // }
-      } catch (err) {
-        console.error(err)
-      }
-    })
-  }
-
```
Comment (starpit) [inline plugins/plugin-wrk/src/lib/chart.ts]: why did you remove this whole block of code?

Comment (jesusprubio) [reply plugins/plugin-wrk/src/lib/chart.ts]: Readded

Comment (starpit) [reply plugins/plugin-wrk/src/lib/chart.ts]: it still shows up as removed for me

Comment (starpit) [reply plugins/plugin-wrk/src/lib/chart.ts]: re-requesting changes

Comment (jesusprubio) [reply plugins/plugin-wrk/src/lib/chart.ts]: re-readded xD
